src/components/Navbar/components/CurrentLocationWidget/CurrentLocationWidget.tsx

```diff
@@ -0,0 +1,71 @@
+import ContentLoader from 'react-content-loader';
+import { Weather } from 'models/Weather';
+import { Container, WeatherIcon, Wrapper, Description, DegreeSign } from './CurrentLocationWidget.styled';
+
+const CurrentLocationWidget = () => {
+  let isLoading = false;
+  const data: Weather | any = {}; // temporary solution in order not to use
+
+  const renderContent = () => {
+    if (isLoading) {
+      if (screen.width > 600) {
+        return (
+          <ContentLoader
+            speed={2}
+            width={219}
+            height={40}
+            viewBox="0 0 219 40"
+            backgroundColor="#e1e1e1"
+            foregroundColor="#ececec"
+          >
+            <circle cx="20" cy="20" r="20" />
+            <rect x="50" y="3" rx="8" ry="8" width="169" height="34" />
+          </ContentLoader>
+        );
+      } else {
+        return (
+          <ContentLoader
+            speed={2}
+            width={120}
+            height={40}
+            viewBox="0 0 120 30"
+            backgroundColor="#e1e1e1"
+            foregroundColor="#ececec"
+          >
+            <circle cx="30" cy="15" r="15" />
+            <rect x="50" y="0" rx="8" ry="8" width="70" height="30" />
+          </ContentLoader>
+        );
```

Comment:
**nitick** I think that we should stay consistent with creating separate files for loaders